docs/sources/tempo/configuration/grafana-alloy/tail-sampling/enable-tail-sampling.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Enable tail-based sampling
+menuTItle: Enable tail-based sampling
+description: Use tail-based sampling to optimize sampling decisions.
+weight:
+---
+
+# Enable tail-based sampling
+
+Tempo provides an inexpensive solution that aims to reduce the amount of tail-based sampling required.
+However, sometimes constraints make a lower sampling percentage necessary or desirable,
+such as runtime or egress traffic related costs.
+Probabilistic sampling strategies are easy to implement,
+but also run the risk of discarding relevant data that you'll later want.
+
+Tail-based sampling works with Grafana Alloy.
+Alloy configuration files are written in [Alloy configuration syntax](https://grafana.com/docs/alloy/latest/concepts/configuration-syntax/).
```

Comment:
```suggestion
Alloy configuration files are written in [Alloy configuration syntax](https://grafana.com/docs/alloy/<ALLOY_VERSION>/concepts/configuration-syntax/).
```

Reply:
I see these docs here: https://grafana.com/docs/tempo/latest/configuration/grafana-alloy/tail-based-sampling/ - I am just wondering about the strategy of pulling it out or duplicating it in tempo docs?